docs/CONFIG.md

```diff
@@ -5035,6 +5075,16 @@ Depending on `SelectionMode`, this represents a difference in the number of bloc
 
 Set to 0 to disable this check.
 
+### SwitchToBestNodeInterval
+```toml
+SwitchToBestNodeInterval = '0s' # Default
+```
+SwitchToBestNodeInterval interval in which to check and force internal services to use the "best" node as
```

Comment:
Would it make sense to think of this as a 'lease' for how long the best node 'sticks'? -`BestNodeLeaseDuration`?